### ISSUE TITLE
Frontmatter missing in raw markdown view

### ISSUE THREAD

**Issue by vondravl** [simov/markdown-viewer#226]
Hi, not sure if my question is not out of scope of this extension. Anyway I thought I'll try to ask. I wonder if it would be possible to view this part of md file. Right now it is skipped. In Obisidian it is properties part.

```
---
category: 
author: 
title: 
source: 
published:
topics: 
tags: 
---
```

**Comment by simov** [comment]
You can switch to the raw markdown at any time and have a look at the file metadata. Also the `title` property is being used to set the document title of your rendered markdown file. 

Other than that I have no idea where it should be the most appropriate place to render the rest of those properties since right now the extension has only a sidebar for the table of contents, and the rest is just the content itself.

**Comment by vondravl** [comment]
Could you just advice me how to switch between Raw and Markdown? Trying to find that, so far I've done that by viewing page source.

**Comment by simov** [comment]
First pin the extension to you toolbar for easier access and then:

![view-raw-md](https://github.com/simov/markdown-viewer/assets/1694112/d88f2995-aa94-48a9-8e1f-203abd754b51)

**Comment by vondravl** [comment]
Actually I knew about that. But that is not showing properties either. Having this text:

```
---
category:
author:
title: 
source: 
clipped: 
published: 
topics: 
tags: 
---

First paragraph of text
```

This is what I see:
![image](https://github.com/simov/markdown-viewer/assets/7070552/62a12de2-d813-47e5-a147-34e12555c85e)
So Properties are skipped as well as in html view. 

Or is it broken only on my side?

**Comment by simov** [comment]
No, you just found a bug, congratulations, it should have been there. It will be fixed in the next release.

**Comment by vondravl** [comment]
Thank you very much. 

Think about some buymeacoffee or support development of the extension page. I think I won't be alone who would like to support you for work done ;)

**Comment by simov** [comment]
This issue got fixed in v5.3, also now the raw markdown view is syntax highlighted if you have the `syntax` option enabled in the Content tab.
